kubernetes/kubernetes.go

```diff
@@ -65,6 +66,19 @@ type OSClientInterface interface {
 	UpdateProject(project string, jsonPatch string) (*osproject_v1.Project, error)
 }
 
+// GetAllServicesByLabels fetches and returns all services in the whole cluster
+// that match the optional labelSelector. This is using the cluster-wide call
+// to fetch the services. The client will need to be created with an account that
+// has cluster-wide privileges to list services.
+func (in *K8SClient) GetAllServicesByLabels(labelsSelector string) ([]core_v1.Service, error) {
```

Comment:
Maybe name this more explicitly?  `GetClusterServicesByLables()` ?   Perhaps it could be a convention for Cluster-wide operations.  I think `All` is a little vague.

Reply:
I was using the "All" naming like in k8s docs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#list-all-namespaces-service-v1-core

But yes, using `Cluster` make a lot more sense. I renamed.